platform/mellanox/docker-syncd-mlnx-rpc.mk

```diff
@@ -11,5 +11,5 @@ endif
 
 $(DOCKER_SYNCD_MLNX_RPC)_CONTAINER_NAME = syncd
 $(DOCKER_SYNCD_MLNX_RPC)_RUN_OPT += --net=host --privileged -t
-$(DOCKER_SYNCD_MLNX_RPC)_RUN_OPT += -v /host/machine.conf:/etc/machine.conf
+$(DOCKER_SYNCD_MLNX_RPC)_RUN_OPT += -v /host/machine.conf:/host/machine.conf
```

Comment:
@marian-pritsak , if you are sure you do not use /etc/machine.conf on your platform, you can remove it.